<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>delete_course</class>
 <widget class="QDialog" name="delete_course">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除特定学生课程</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_sname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>学生姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_cname">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>40</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>课程名称</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_sname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_cname">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>delete_course</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>delete_course</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
